apps/website/src/app/docs/report/page.tsx

```diff
@@ -0,0 +1,232 @@
+"use client";
+
+import {
+  CheckResultCode,
+  ConsolidatedNameGuardReport,
+  Normalization,
+  Rating,
+} from "@namehash/nameguard";
+import { ReportBadge, ReportIcon } from "@namehash/nameguard-react";
+
+export default function RatingShieldsPage() {
+  return (
+    <div className="py-12 max-w-7xl mx-auto px-6 space-y-12">
+      <div className="divide-y divide-gray-100">
+        <h1 className="justify-center flex font-bold text-2xl my-8">
+          {"<"}ReportIcon {"/>"} documentation
+        </h1>
+
+        <div className="divide-y divide-gray-100">
+          <div className="grid grid-cols-6 gap-x-6 py-5 text-center font-medium">
+            <div></div>
+            <div>Success</div>
+            <div>Warn</div>
+            <div>Alert</div>
+            <div>Unknown</div>
```

Comment:
Why don't the "Unknown" `ReportIcon` have tooltips? The "Unknown" `ReportBadge` have a tooltip and the `ReportIcon` should work exactly the same.

Additionally, these should all have "clicky-hand" cursors as they are all clickable and they all should open when clicked.

Even if we have an "Unknown" result inside this UI component, clicking on it should work correctly no matter what.